src/main/scala/io/citrine/lolo/validation/Visualization.scala

```diff
@@ -0,0 +1,133 @@
+package io.citrine.lolo.validation
+
+import io.citrine.lolo.PredictionResult
+import org.apache.commons.math3.distribution.CauchyDistribution
+import org.knowm.xchart.XYSeries.XYSeriesRenderStyle
+import org.knowm.xchart._
+import org.knowm.xchart.internal.chartpart.Chart
+
+/**
+  * Visualization on predicted vs actual data of type T
+  */
+trait Visualization[T] {
+  /**
+    * Produce a visualization, as a [[Chart]], from predicted-vs-actual data
+    */
+  def visualize(data: Iterable[(PredictionResult[T], Seq[T])]): Chart[_, _]
+}
+
+/**
+  * Histogram of the error divided by the predicted uncertainty
+  * @param nBins number of bins in the histogram
+  * @param range of the horizontal axis, e.g. x \in [-range/2, range/2]
+  */
+case class StandardResidualHistogram(nBins: Int = 128, range: Double = 8.0) extends Visualization[Double] {
```

Comment:
I can imagine the number of these visualization classes could get quite large. 

At this point I would argue what I'm about to propose isn't worth it, but I wonder if we could create a more compassable API for generating plots that wouldn't require someone to define a `Visualization` class from scratch every time they wanted to make a small tweak.

Reply:
I think "tweak"-like changes should be codified in the state of the `Visualization`, e.g. `nBins`.  The charts that are emitted are also editable, so a user could "compose" a new one by building another chart inside of it and then tweaking it before returning it.

Reply:
True. Tweaking the plot that's generated could cover most use cases. This solution is fine with me.